---
output:
  word_document:
    reference_docx: myref.docx
bibliography: MyBib.bib
csl: Evolution.csl

---

```{r setup, include = F}

rm(list=ls())

#Load knitr
library(knitr)

#Set some options
options(na.action='na.fail')
opts_knit$set(root.dir = '..')
opts_chunk$set(echo=FALSE,
               warning=FALSE,
               message=FALSE,
               fig.path='../Figures/',
               fig.width= 5,
               fig.height= 4,
               fig.show='asis',
               dev='png')

```

```{r load and clean, include = F}

#Load and clean data
library(knitr)
library(ggplot2)
library(plyr)
library(maps)
library(plotrix)
library(reshape)
library(qqman)
source('R_scripts/GTH_FUNCTIONS.R')

```

**Genomic insights into continental-scale population structure and adaptation in a model passerine**

Lewis G. Spurgin,^1^ Mirte Bosse^2^, Veronika Laine^2^, Kees van Oers^2^, The Great Tit Hap Map Consortium, Ben C. Sheldon^1^, Martien A. M. Groenen^3^, Marcel E. Visser^2,3^, Jon Slate^4^


^1^ Edward Grey Institute, University of Oxford, United Kingdom  
^2^ Department of Animal Ecology, Netherlands Institute of Ecology (NIOO-KNAW), Wageningen, the Netherlands  
^3^ Animal Breeding and Genomics Center, Wageningen University, the Netherlands  
^4^ Department of Animal and Plant Sciences, University of Sheffield, United Kingdom

**Abstract**

Despite massive increases in the availability of DNA sequence data, determining the genetic basis of adaptation in natural poulations has proved difficult. Thus in many cases identifying genes under selection in natural populations requires i) large numbers of individuals, ii) large numbers of molecular markers, and iii) excellent genomic resources. Here, using a panel of ~500,000 SNPs and samples from 23 populations, we disentangle the the roles of demographic and adaptive processes in shaping genomic variation in the European Great tit. The great tit is a widespread passerine bird that inhabits a wide range of ecological environments, and is a  model species for ecological and behavioural research. We find that patterns of genomic diversity are consistent with recent spread and admixture from multiple southern refugia. While some structure exists between southern and northern populations, we find that the majority of northern and central European great tits exist as a single, panmictic population. Selection analyses reveal clear evidence for genomic regions under selection between southern and northern European populations, as well as within the northern panmictic cluster. We identify candidate genes for adaptation to temperature and latitude. Our results provide novel insights into ecological adaptation occurs in .

**Introduction**
Identifying the genetic basis of adaptation is one of the major aims of evolutionary biology. 

It is becoming increasingly clear that in order to properly understand the genetic basis of ecological adaptation, we require both very large numbers of genetic markers and many individuals, as well as a well-annotated genome. Unfortunately however it has not been possible to generate data on this scale in a wild animal species.

The European great tit (*Parus major major*) is...

In this study we analyse genomic variation across 23 European great tit populations. Our aims were twofold. First, we quantify levels of fenetic variation and population structure among European great tit populations, and relate this to hypotheses of colonisaiton history. Second, we identify regions of the genome involved in ecological adaptation, with the aim of identifying biological processes involved in adaptation across this species' range.

**Materials and methods**


*Sampling and molecular methods*

Samples were obtained from etc.

DNA extraction details

All samples were genotyped using 485,122 SNPs.

*Statistical analyses*

Phasing was carried out using SHAPEIT version....

For all subsequent analyses we removed related individuals based on pedigree information, and based on marker relatedness. To calculate the latter we used a pruned dataset, removing SNPs from the z chromosome, chromosome 1A (which was involved in a chromosome-wide inversion - see results), and SNPs in linkage disequilibrium (LD), based on a threshold of VIF = 1.2 (corresponding to a correlation coefficient of XX). This pruning left is with 324,471 SNPs. We then estimated identity-by-descent (IBD) of all pairs of individuals within populations using PLINK v1.9 [@Purcell2007], and based on the distribution of IBD values (Fig. SXX), randomly removed individuals from dyads with relatedness values greater than 0.4.

To visualise overall patterns of genetic diversity within and among populations, we calculated individual inbreeding coefficients from the pruned SNP data, using the methods of moments estimator implemented in PLINK. Then, using unpruned data, we estimated pairwise LD between markers separately for each population. For this analysis we used VCFtools v0.1.14 [@Danecek2011], and due to the very large number of pairwise comparisons between markers we only calculated LD between markers within 300kb of one another. Finally, using the phased data, we calculated haplotype diversity and haplotype block lengths for each population.

To quantify population structure, we used the Bayesian algorithm implemented in software ADMIXTURE [@Alexander2009]. The analysis was run nine times, varying the number of assumed genetic clusters (K) from two to 10, and cross-validation error was calculated for each value of K as an estimate of model fit.  As an additional visualisation of population structure, we used multidimensional scaling (MDS) analysis, implemented in PLINK. Both of these analyses were implemented using the pruned dataset.

We used two approaches to identify regions of the genome under selection across European great tit populations. First, we identified outlier regions of high differentiation among genetic clusters (identified using ADMIXTURE), using EigenGWAS [@Chen2015]. This approach uses eigenvectors as 'phenotypes' in a genome-wide association study, and thus identifies outlier SNPs of highe genetic differentiation, while inherently controlling for genome-wide population structure. Nonetheless, we also used a more established approach based on F^ST [BAYESCAN ] to test for outlier SNPs between genetic clusters.

Second, using individuals from a single, widespread genetic cluster (see results) we used BAYESCENV - an FST-based approach to identify regions of the genome significantly associated with ecological variation. For this analysis we used DETAILS and autumn temperature as the environmental variable. Outlier regions from BAYESCENV were confirmed using latent factor mixed models, implemented in the program LMFF.


**Results**

In total, XX birds were genetyped at between YY and ZZ SNPs. Details of population-level sample sizes and SNP success rates are provided in Table S1.

*Genetic diversity and population structure*

Differences in levels of genetic diversity were predominantly between southern and northern populations, with lowest levels of genetic diversity in Corsican and Spanish populaitons (Fig. 1). Levels of genetic diversity among central and northern European populations were almost identical (Fig. 1).

Linkage disequilibrium declined very rapidly, to background levels of XX-YY, in all populations (Fig. 2). However, there were significant differences in the initial rate of LD decay (), as well as in levels of long-range LD (). At very short inter-SNP distances, highest levels of LD were found in Corsica, Spain and Scotland (Fig. 2), and LD in Corsica decayed less rapidly than in all other populations.

Admixture analyses revealed that three genetic clusters best explained the great tit data, with one cluster corresponding to the Corsican population, one to the two Spanish populaitons and the third at high prevalence in all remaining populations (Fig. 3). The Corsican and Spanish clusters were also found in the remaining European populations, and admixture proportions were extremely similar across all populations apart from Spain and Corsica (Fig. 3). Using K = 4 separated out the French and UK populations from the rest of Europe, while higher values of K did not correspond to geographical distribution (Fig. SXX). Multi-dimensional scaling analysis confirmed the results of structure, with the first four dimensions separating Corsica, Spain, France and the UK, respectively (Fig. SXX). We found no evidence for a pattern of isolation by distance, regardless of whether we included all populations or just the large Euopean cluster (Fig. SXX).


*Selection*



**Discussion**
NOTES

Finestructure - I suggest we try and get hold of some samples from Italy and the Balkans, then run FineStructure using a more comprehensive dataset as a separate phylogeography paper.

Cluctch size and lay date - We discussed using these as the environmental variable in BAYESCENV analyses - however, these data are very patchy in the field data files. What is the best way to get accurate and comparable estimates for these variables?


**Figures**

**Figure 1**
```{r Figure 1}
het <- read.table('Plink/PmajorEurope/AUT/PmajorEurope.het',header=T)
se <- function(x) mean(x/sqrt(length(x)))


ddx <- ddply(het,
             .(FID),
             summarise,
             meanF = mean(F,na.rm=T),
             seF = se(F))
ddx <- subset(ddx,FID!='Oosterhout')
ddx$FID <- factor(ddx$FID,levels = ddx$FID[order(ddx$meanF)])

ggplot(ddx,aes(y = meanF,x = FID)) +
  geom_point() +
  geom_pointrange(aes(ymax = meanF+seF,ymin = meanF-seF))+
  theme_lgs()+
  theme(axis.text.x = element_text(angle=60,hjust=1),
        legend.position= 'none')+
  xlab('') +
  ylab('F')

```

**Figure 2**

**Figure 3**

**Figure 4**


**References**


